engine_graphics/BUILD

```diff
@@ -1,35 +1,45 @@
 cc_library(
     name = "engine_graphics",
-    srcs = glob([
-        "src/**/*.cpp",
-    ],
-    exclude = ["src/renderer_test_node.cpp"],
+    srcs = glob(
+        [
+            "src/**/*.cpp",
+        ],
+        exclude = ["src/renderer_test_node.cpp"],
     ),
     hdrs = glob([
         "include/**/*.hpp",
         "include/**/*.h",
     ]),
+    copts = ["-Iexternal/nlohmann/include"],
     includes = ["include"],
     visibility = ["//visibility:public"],
     deps = [
         "//engine_datamodel:engine_datamodel_math",
         "//engine_datamodel:engine_datamodel_rendering",
         "//engine_datamodel:engine_datamodel_visual",
-        "@nlohmann",
+        "//gl_extensions",
     ],
-    copts = ["-Iexternal/nlohmann/include"],
 )
 
 cc_binary(
     name = "renderer_test_node",
     srcs = ["src/renderer_test_node.cpp"],
+    data = ["//parsers:x3d_test_resources"],
+    linkopts = [
+        "-lpthread",
+        "-lOpenGL",
+        "-lGL",
+        "-lGLU",
+        "-lglut",
+        "-lGLX",
+        "-lGLEW",
+    ],
+    visibility = ["//visibility:public"],
     deps = [
         ":engine_graphics",
+        "//input_handler:engine_input_handler",
```

Comment:
Input handler should be either removed, or designed and implemented properly, which is out of this task's scope